src/foam/lib/json/JSONParser.java

```diff
@@ -12,7 +12,7 @@
 public class JSONParser
   extends foam.core.ContextAwareSupport
 {
-  private Parser   parser   = new FObjectParser();
+  private Parser   parser   = new ExprParser();
```

Comment:
Use protected instead of private. Makes easier to test and debug. Why the change?